ycmd/completers/go/tests/gocode_completer_test.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env python
```

Comment:
If you look inside `get_completions_tests.py` you'll find E2E tests for every completion engine :+1:  https://github.com/Valloric/ycmd/blob/master/ycmd/tests/get_completions_test.py#L669 this is the basic one for python.